resources/npm-git.sh

```diff
@@ -9,7 +8,8 @@
 # From: https://github.com/graphql/graphql-js/blob/master/resources/npm-git.sh
 
 BUILD_DIR=latest
-
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
can you uncommit that change please? This will break our CI...

Reply:
Thanks for catching that. 🥇